notebook/gateway/handlers.py

```diff
@@ -18,8 +18,8 @@
 from ipython_genutils.py3compat import cast_unicode
 from jupyter_client.session import Session
 from traitlets.config.configurable import LoggingConfigurable
-
 from .managers import GatewayClient
+from ..db_util import ExecuteQueries
```

Comment:
can we use Absolute Imports here?

Reply:
tried but this will not wok because of the way Jupyter Notebook is initialising inside Jupiter server.